_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -188,6 +220,36 @@ This `img` element has an [explicit role][] of `none`. However, it is [focusable
 <img role="none" tabindex="0" src="/test-assets/shared/w3c-logo.png" />
 ```
 
+#### Failed Example 6
+
+This `img` element displays a CSS `background-image`, has an empty [accessible name][] and an [implicit role][] of `img` because it is missing 
+an empty `alt` attribute.
```

Comment:
I am not sure that "because it is missing an empty `alt` attribute" is really needed in the description (maybe it is instead missing an `aria-label` attribute to give it an accessible name).
```suggestion
This `img` element displays a CSS `background-image`, has an empty [accessible name][] and an [implicit role][] of `img`.
```